### PR TITLE
Route WEB_FEATURES.yml changes to web-features contributors

### PR DESCRIPTION
**Pull request by foolip**
Fixes https://github.com/web-platform-tests/wpt-pr-bot/issues/253.

### CODE REVIEW

**Comment by foolip** [comment]
This depends on https://github.com/web-platform-tests/wpt-pr-bot/pull/254 and I'd like to rebase it after that lands.

File: test/find-owners.js
Comment (foolip) [inline test/find-owners.js:82]: In other words, if a PR touches both `WEB_FEATURES.yml` and tests, it gets routed only to the regular reviewers. I think this is best for the long term, that we don't expect web-features maintainers to review every such change. We just want to reduce the noise from changes that are purely web-features.

Comment (jcscottiii) [reply test/find-owners.js:82]: This makes sense for now. And we can always improve the linting (for example, linting based on the vendored data.json) to help regular reviewers have confidence in those changes.

**Review by jcscottiii — APPROVED** [review]

File: test/find-owners.js
Comment (foolip) [reply test/find-owners.js:82]: Yeah, that would be good even if you and I are the only reviewers, because we'll never catch all minor typos :)

**Pull request merged**
